requirements.txt

```diff
@@ -1,42 +1,48 @@
-certifi==2022.6.15
+#
+# This file is autogenerated by pip-compile with Python 3.8
+# by the following command:
+#
+#    pip-compile requirements.in
+#
+--index-url https://nexus.dev.holmes.nl/repository/pypi-all/simple
```

Comment:
I used to remove this path to the Nexus manually after `pip-compile`, since people on "the Internet" don't have many use for it. Perhaps it might even break for them if they do `pip install -r requirements.txt`. There also don't seem to be any internal tools in this `requirements.txt`.

On the other hand it would involve an extra action when updating dependencies.

I think it could be removed safely though. Everyone in the organisation probably has it in their `pip.ini` already.

Reply:
You can run `pip-compile` with `--no-emit-index-url --no-emit-options` to keep it from adding local configuration to the `requirements.txt` file.

Reply:
good catch!
I just found out that `tox -e upgrade` contains the pip compile `--no-header --no-emit-options` flags. So that is the way to update requirements.txt in the future.